stetho/src/main/java/com/facebook/stetho/common/android/FragmentApi.java

```diff
@@ -0,0 +1,459 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.common.android;
+
+import android.app.Activity;
+import android.view.View;
+
+import com.facebook.stetho.common.ClassUtil;
+import com.facebook.stetho.common.LogUtil;
+import com.facebook.stetho.common.Util;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+// TODO: don't use reflection to implement the accessors
```

Comment:
Please provide a little more context.  Specifically, there are two issues at play here:

1. The support library is needed as an optional/provided dependency, but we cannot express that today with the `maven` plugin in Gradle (I believe we need to upgrade to `maven-publish`.
2. The class needs to be divided significantly so that the Honeycomb+ APIs can be separated in terms of the class verifier from the Support library APIs.  Once separated, we can use booleans that use `Class.forName` to detect the presence of the classes we're interested in and adjust behaviour accordingly.

That said, I'm fine proceeding as-is per our conversation offline.